frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/enterprise.embedding.unit.spec.tsx

```diff
@@ -53,12 +57,44 @@ describe("[EE, no token] embedding settings", () => {
       });
     });
 
+    describe("embedding SDK", () => {
+      it("should show info about embedding SDK", async () => {
+        const withinEmbeddingSdkCard = within(
+          screen.getByRole("article", {
+            name: "Embedding SDK for React",
+          }),
+        );
+
+        expect(
+          withinEmbeddingSdkCard.getByRole("heading", {
+            name: "Embedding SDK for React",
+          }),
+        ).toBeInTheDocument();
+        expect(
+          withinEmbeddingSdkCard.getByText(
+            /Interactive embedding with full, granular control./,
+          ),
+        ).toBeInTheDocument();
+      });
+
+      it("should allow access to the embedding SDK settings page", async () => {
+        // Go to embedding SDK settings page
+        await userEvent.click(
+          within(
+            screen.getByRole("article", {
+              name: "Embedding SDK for React",
+            }),
+          ).getByRole("button", { name: "Try it out" }),
+        );
+
+        expect(
+          screen.getByLabelText("Cross-Origin Resource Sharing (CORS)"),
+        ).toBeDisabled();
```

Comment:
Why do we expect the CORS label text to be disabled?

Reply:
by our convention,
`common.*` tests OSS
`enterprise.*` tests EE binary without token e.g. cloud starter
`premium.*` tests EE binary with token e.g. Pro/Enterprise users

These suite names are confusing indeed confusing for first-timers. If you have any suggestion,s feels free to open an RFC to discuss this.

Reply:
@WiNloSt I see, the fact that "enterprise" is without token did surprise me, as I thought that "enterprise" is the higher tier than "premium" (I thought Premium is kinda like Pro). I might open a RFC to rename that to be more explicit.